doc/api/timers.md

```diff
@@ -2,87 +2,127 @@
 
     Stability: 3 - Locked
 
-All of the timer functions are globals.  You do not need to `require()`
-this module in order to use them.
+The `timer` module exposes a global API for scheduling callback functions to
+execute at some future period of time. Because the timer functions are globals,
```

Comment:
execute is "one-off", period would rather indicate a sequence of time, no? Anyhow this sounds a little bulky.